x-pack/plugins/cases/public/plugin.ts

```diff
@@ -35,9 +38,14 @@ export class CasesUiPlugin implements Plugin<void, CasesUiStart, SetupPlugins, S
     return {
       canUseCases: canUseCases(core.application.capabilities),
       getCases: getCasesLazy,
+      getCasesContext: getCasesContextLazy,
       getRecentCases: getRecentCasesLazy,
       getCreateCaseFlyout: getCreateCaseFlyoutLazy,
       getAllCasesSelectorModal: getAllCasesSelectorModalLazy,
+      getCreateCaseFlyoutNoProvider: getCreateCaseFlyoutLazyNoProvider,
```

Comment:
Temporary method. This will replace getCreateCaseFlyout after I remove it from the timelines plugin in further PRs.